src/components/buttons.js

```diff
@@ -5,37 +5,39 @@ import { SvgIcon } from './SvgIcon';
 /**
  * @typedef ButtonProps
  * @prop {import('preact').Ref<HTMLButtonElement>} [buttonRef]
- * @prop {import("preact").ComponentChildren} [children]
- * @prop {string} [className]
  * @prop {string} [icon] - Name of `SvgIcon` to render in the button
  * @prop {'left'|'right'} [iconPosition] - Icon positioned to left or to
  *   right of button text
- * @prop {boolean} [disabled]
  * @prop {boolean} [expanded] - Is the element associated with this button
  *   expanded (set `aria-expanded`)
  * @prop {boolean} [pressed] - Is this button currently "active?" (set
  *   `aria-pressed`)
- * @prop {(event: Event) => void} [onClick]
  * @prop {'small'|'medium'|'large'} [size='medium'] - Relative button size:
  *   affects padding
- * @prop {Object} [style] - Optional inline styles
- * @prop {string} [title] - Button title; used for `aria-label` attribute
```

Comment:
I would recommend keeping this line. It points to a different use between our custom button and the default ones.

Reply:
will do, makes sense